apps/examples/wifi_manager_sample/wm_test.c

```diff
@@ -860,65 +863,65 @@ void wm_auto_test(void *arg)
 		wm_display_state(NULL);
 
 		/* Start STA mode */
-		printf("Start STA mode\n");
+		printf("[WT] Start STA mode\n");
 		res = wifi_manager_set_mode(STA_MODE, NULL);
 		if (res != WIFI_MANAGER_SUCCESS) {
-			printf(" Set STA mode Fail\n");
+			printf("[WT]  Set STA mode Fail\n");
```

Comment:
I think "[WT]" needs a consistent space behind it.

Reply:
Thanks I'll fix it